.github/workflows/ci.yml

```diff
@@ -63,9 +63,7 @@ jobs:
     strategy:
       matrix:
         rust:
-          - stable
           - beta
-          - 1.62.0
```

Comment:
I don't think this will be acceptable, if I'm honest. The conditional compile attribute seems like a better long-term solution. Ultimately, I _do_ want to run unit tests on this entire matrix, even if there are none at present.

Reply:
I don't know many about Github CI config, so is this going to work:
```yaml
    name: Test
    runs-on: ubuntu-latest
    needs: [checks, lints]
    strategy:
      matrix:
        rust:
          - stable
          - beta
          - 1.62.0
    if: matrix.rust == "beta"
    steps:
...

```

Reply:
No, I believe it's more a matter of using an attribute like this in the test: https://github.com/dtolnay/ref-cast/blob/f7b0b87cacb4cc0866a07150205e6d35e634fb23/tests/compiletest.rs#L1 exchanging `beta` for `nightly`. The CI stuff can be left as it was.

Reply:
With `rustversion` crate we can set `no_display_no_impl` test to run only since 1.68.0, which is the version I've found to be first with such syntax as stable and beta. So all tests will run in all toolchains and `no_display_no_impl` only in stable/beta